.gitignore

```diff
@@ -9,3 +9,4 @@ coverage/
 .env
 bundle.js
 bundle.js.map
+config.js
```

Comment:
Can you leave a demo skeleton of what is in the config file. If it contains credentials and settings that need to be passed into the deployed app, put them in .env so they will be run with the Docker images in production.